test/unit/spec/EnrollTotpController_spec.js

```diff
@@ -689,319 +529,600 @@ function (Okta, OktaAuth, LoginUtil, Util, DeviceTypeForm, BarcodeForm,
+      itp('displays the correct factorBeacon for Okta Verify', function () {
+        return setupOktaTotp().then(function (test) {
+          expect(test.beacon.isFactorBeacon()).toBe(true);
+          expect(test.beacon.hasClass('mfa-okta-verify')).toBe(true);
+        });
+      });
+      itp('displays the correct factorBeacon for Google Authenticator', function () {
+        return setupGoogleTotp().then(function (test) {
+          expect(test.beacon.isFactorBeacon()).toBe(true);
+          expect(test.beacon.hasClass('mfa-google-auth')).toBe(true);
+        });
+      });
+      itp('has a "back" link in the footer', function () {
+        return setupOktaTotp().then(function (test) {
+          Expect.isVisible(test.form.backLink());
         });
       });
     });
 
+    Expect.describe('Enroll factor', function () {
+      testEnrollFactor(setupOktaTotp, setupAndEnrollOktaTotp, setupGoogleTotp, setupAndEnrollGoogleTotp, 'testStateToken');
+    });
+
+    Expect.describe('Enroll factor on Idx Pipeline', function () {
+      testEnrollFactor(setupOktaTotpWithIdxPipeline, setupAndEnrollOktaTotpWithIdxPipeline, setupGoogleTotpWithIdxPipeline, setupAndEnrollGoogleTotpWithIdxPipeline, '01testStateToken');
+    });
+
+    Expect.describe('Scan qrcode', function () {
+      testScanQRCode(setupAndEnrollOktaTotp, setupOktaTotp, resTotpEnrollSuccess, resAllFactors);
+
+      Expect.describe('Manual setup', function () {
+        testScanQRCodeManualSetup(setupAndEnrollOktaTotp, setupOktaTotp, resTotpEnrollSuccess, 'testStateToken');
+      });
+
+      Expect.describe('Pass code form', function () {
+        testScanQRCodePassCodeForm(setupAndEnrollOktaTotp, setupOktaTotp, resTotpEnrollSuccess, 'testStateToken');
+      });
+    });
+
+    Expect.describe('Scan qrcode on Idx Pipeline', function () {
+      testScanQRCode(setupAndEnrollOktaTotpWithIdxPipeline, setupOktaTotpWithIdxPipeline, resFactorEnrollTotpEnrollSuccess, resFactorEnrollAllFactors);
+
+      Expect.describe('Manual setup on Idx Pipeline', function () {
+        testScanQRCodeManualSetup(setupAndEnrollOktaTotpWithIdxPipeline, setupOktaTotpWithIdxPipeline, resFactorEnrollTotpEnrollSuccess, '01testStateToken');
+      });
+
+      Expect.describe('Pass code form on Idx Pipeline', function () {
+        testScanQRCodePassCodeForm(setupAndEnrollOktaTotpWithIdxPipeline, setupOktaTotpWithIdxPipeline, resFactorEnrollTotpEnrollSuccess, '01testStateToken');
+      });
+    });
+
+    Expect.describe('Okta Verify with Push', function () {
+      testOktaVerify(setupAndEnrollOktaPush, setupOktaPush, resActivatePushSms, resActivatePushTimeout, 'testStateToken');
+    });
+
+    Expect.describe('Okta Verify with Push on Idx Pipeline', function () {
+      testOktaVerify(setupAndEnrollOktaPushWithIdxPipeline, setupOktaPushWithIdxPipeline, resFactorActivatePush, resFactorEnrollActivatePushTimeout, '01testStateToken');
+    });
+
+    Expect.describe('Manual setup', function () {
```

Comment:
this one should be inside `Expect.describe('Okta Verify with Push'` right? it is the manual setup for that